modules/gltf/gltf_document.cpp

```diff
@@ -3258,13 +3316,26 @@ GLTFTextureIndex GLTFDocument::_set_texture(Ref<GLTFState> state, Ref<Texture2D>
 	return gltf_texture_i;
 }
 
-Ref<Texture2D> GLTFDocument::_get_texture(Ref<GLTFState> state, const GLTFTextureIndex p_texture) {
+Ref<Texture2D> GLTFDocument::_get_texture(Ref<GLTFState> state, const GLTFTextureIndex p_texture, bool p_is_normal) {
```

Comment:
Do we also require an argument for p_is_roughness for ORM / roughness maps and passing in the associated normal map?

Godot uses the roughness flag to enable the roughness limiter on that texture channel.

Reply:
The roughness definition is only known when we have a material.

As far as I know the only important distinctions is the source_color, and the normals image types other than regular images.

Reply:
glTF files have a material. Therefore we should be able to know which channels and textures are used for normals and roughness, same way we know something is a normal map

Reply:
I'll try to implement this after ResourceLoader::import is done.

Reply:
I added an enum but I didn't see any cases.